### ISSUE TITLE
Negative z-index integer gets converted to positive integer 

### ISSUE THREAD

**Issue by darwinshameran** [jaredh159/tailwind-react-native-classnames#257]
According to [Tailwind docs](https://tailwindcss.com/docs/z-index#using-negative-values) you can use negative values such as `-z-50` as the z-index, however twrnc seems to always convert it to a positive value. I'm not sure if this is intended behavior or a bug:

```
console.log(tailwind.style('px-4 -z-1'))
{"paddingLeft": 16, "paddingRight": 16, "zIndex": 1}
```

**Comment by jaredh159** [comment]
thanks for letting me know, fix coming in a few minutes, in version `3.6.6`.

**Comment by jaredh159** [comment]
patched release is out 👍
